app/grandchallenge/reader_studies/static/js/reader_studies/interface_choice.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function() {
```

Comment:
Could this be done with htmx insead?

Reply:
Looking into that now. I haven't used it before.

Reply:
Anne added it and did some really nice things with the Challenge Request forms. Take a look there but this looks like the example we'd need https://htmx.org/examples/value-select/. You should speak with Anne about it but she is off today.